includes/Core/Authentication/Authentication.php

```diff
@@ -1277,4 +1277,28 @@ private function verify_user_input_settings() {
 		}
 	}
 
+	/**
+	 * Filters feature flags using features received from the proxy server.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param boolean $feature_enabled Original value of the feature.
+	 * @param string  $feature_name    Feature name.
+	 * @return boolean State flag from the proxy server if it is available, otherwise the original value.
+	 */
+	private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
+		$transient_name = 'googlesitekit_remote_features';
+		$features       = $this->transients->get( $transient_name );
+		if ( empty( $features ) ) {
+			$features = $this->google_proxy->get_features( $this->credentials );
+			$this->transients->set( $transient_name, $features, DAY_IN_SECONDS );
```

Comment:
noticed that `$features` may be an error here if `google_proxy->get_features( $this->credentials );` returns an error. In that case, we may want to try again more quickly, maybe in an hour?

Reply:
Yep, sounds good to me. Updated to be an hour for errors.